PokeAlarm/Manager.py

```diff
@@ -793,7 +800,9 @@ def process_gym(self, gym):
         gym.update({
             "gym_name": gym_info.get('name', 'unknown'),
             "gym_description": gym_info.get('description', 'unknown'),
-            "gym_url": gym_info.get('url', 'https://raw.githubusercontent.com/RocketMap/PokeAlarm/master/icons/gym_0.png'),
+            "gym_url": gym_info.get(
+                'url',
+                'https://raw.githubusercontent.com/RocketMap/PokeAlarm/master/icons/gym_0.png'),
```

Comment:
What's going on here?

Reply:
Style settings of project says this line was too long